### PR TITLE
tomcat 8 + failOnError flag

### PR DESCRIPTION
**Pull request by raffaeleragni**
Added a needed the flag for JSPs to not stop when there is an error in compilation.
Plus added tomcat 8 support (merely a copy paste of the submodule, changed versions)

### CODE REVIEW

**Comment by leoherbie** [comment]
any update on accepting tomcat 8 support?

File: jspc-compilers/jspc-compiler-tomcat8/pom.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <groupId>org.jasig.mojo.jspc</groupId>
+        <artifactId>jspc-compilers</artifactId>
+        <version>2.0.1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>jspc-compiler-tomcat8</artifactId>
+    <name>JSPC Compiler for Tomcat 8</name>
+
+    <properties>
+        <tomcatVersion>8.0.20</tomcatVersion>
```
Comment (leoherbie) [inline jspc-compilers/jspc-compiler-tomcat8/pom.xml]: I would suggest setting this to the latest tomcat 8 version if/when accepting this PR.

**Comment by timlevett** [comment]
We merged in #22

**Comment by ChristianMurphy** [comment]
@raffaeleragni Could you merge/rebase with the latest code and apply just the `failOnError` changes?

**Comment by ChristianMurphy** [comment]
Hey @raffaeleragni, :wave: 
Thanks for opening taking the time to look into this, unfortunately this PR cannot be merged with the current merge conflicts.
If you have a chance, it would great to get a fresh PR with the fail on error changes applied to latest version.

**Pull request closed without merging**